backend/src/utils/helpers.js

```diff
@@ -34,6 +34,6 @@ exports.decodeUTF8Text = function(s) {
         console.warn(e.message)
         console.warn(s)
         // replacing non ascii characters with spaces seems to work. We could just return s too
-        return s.replace(/[^\x00-\x7F]/g, " ");
+        return s.replace(/[^\x00-\x7F]/g, ' ') // eslint-disable-line
```

Comment:
Din't know you can specify hex characters in regex...

Reply:
I believe nunez did that.

Reply:
@NunoDasNeves that is disappointing to hear.